src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,17 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  if (user === null) {
+    return null;
+  }
```

Comment:
let's remove the check if the user exists here, this component's goal is to render the user info, so let's not increase the complexity and add the check logic here

Reply:
done